posts/Yoann/2017-11-27-introducing-hibernate-search-jsr352-mass-indexing-job.adoc

```diff
@@ -0,0 +1,158 @@
+= Introducing the Hibernate Search JSR-352 mass indexing job
+Yoann Rodière
+:awestruct-tags: [ "Hibernate Search", "JSR-352" ]
+:awestruct-layout: blog-post
+---
+
+Hibernate Search 5.9 (still in beta) features a new integration to JSR-352 runtimes,
+providing a new implementation of mass indexing (indexing a high volume of entities)
+as a JSR-352 job.
+
+Why is it nice? How to use it? Let's find out!
+
+== What's JSR-352?
+
+"JSR 352: Batch Applications for the Java Platform" is a Java specification
+defining interfaces and mechanisms to execute batch processes ("jobs"),
+i.e. processes that involve applying the same operation over and over again to a large number of items.
+
+In JSR-352:
+
+* The job definition is provided as an XML file and a few Java classes, called batch artifacts
+  (in our case, the definition is provided by Hibernate Search).
```

Comment:
just a style suggestion: try avoid using the parenthesis whenever you can?
In this case the sentence works fine by just removing them.